test.js

```diff
@@ -14,6 +14,10 @@ function commit (msg) {
   shell.exec('git commit --allow-empty -m"' + msg + '"')
 }
 
+function execCli (argString) {
+  return shell.exec('node ' + cliPath + ' ' + argString)
```

Comment:
As Steve said, I don't think we need to worry about allowing falsy values here, but this should be fine.